pkg/controller/template/test_data/templates/worker/01-worker-kubelet/aws/units/kubelet.service

```diff
@@ -18,7 +18,7 @@ contents: |
         --container-runtime=remote \
         --container-runtime-endpoint=/var/run/crio/crio.sock \
         --allow-privileged \
-        --node-labels=node-role.kubernetes.io/worker \
+        --node-labels=node-role.kubernetes.io/master,node.openshift.io/os_version=${OS_VERSION} \
```

Comment:
why is the worker label changed to master in the `01-worker-kubelet`?

Reply:
It isn't changing. The comma is adding a another label called node.openshift.io/os_version with the os version. These node labels only get added on kubelet registration, so we will need a daemonset to 'update' the label on RHEL7->RHEL8 upgrades.

Reply:
to clarify it seems to have gone from `/worker` to `/master, node.openshift.io/os....` 

why is that master?

Reply:
good catch. I typoed that.

Reply:
@kikisdeliveryservice got that... transposition... fixed. thanks!

Reply:
np!